src/Avalonia.Controls/Window.cs

```diff
@@ -462,18 +462,19 @@ void SetIsEnabled(IEnumerable<Window> windows, bool isEnabled)
 
         void SetWindowStartupLocation()
         {
+            var scaling = PlatformImpl?.Scaling ?? 1;
             if (WindowStartupLocation == WindowStartupLocation.CenterScreen)
             {
                 var screen = Screens.ScreenFromPoint(Bounds.Position);
 
                 if (screen != null)
-                    Position = screen.WorkingArea.CenterRect(new Rect(ClientSize)).Position;
+                    Position = screen.WorkingArea.CenterRect(new Rect(ClientSize * scaling)).Position;
```

Comment:
unfortunately this doesn't work for me.

<img width="1440" alt="image" src="https://user-images.githubusercontent.com/4672627/47588468-2377b100-d95e-11e8-85ef-3b450eb38430.png">

Reply:
I believe this is because the `ClientSize` is being treated as screen pixels somewhere and isn't being scaled. This is probably a backend-specific bug as I'm not getting it on Win32.

Which backend are you running on?

Reply:
@danwalmsley any further info here?